server.js

```diff
@@ -22,11 +16,106 @@ app.use(cors())
 app.use(express.json())
 
 // Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+
+app.get("/", (req, res) => {
+  res.send(
+    "Hello This is the root-endpoint to my netflix API. By adding /endpoints to the url you can get an overview of the possible endpoints"
+  )
+})
+
+app.get("/endpoints", (req, res) => {
+  res.send(listEndpoints(app))
+})
+
+app.get("/netflix-titles", (req, res) => {
+  const { title } = req.query
+
+  let titles = netflixData
+
+  if (title) {
+    titles = titles.filter(
+      (item) => item.title.toLowerCase().indexOf(title.toLowerCase()) !== -1
+    )
+  } else {
+    res.status(200).json({
+      response: netflixData,
+      success: true,
```

Comment:
You use error handling quite consistently throughout your API. Good job!

Reply:
Thank you for the review, Isabel!